x-pack/plugins/security_solution/public/management/common/translations.ts

```diff
@@ -215,6 +215,66 @@ export const CONSOLE_COMMANDS = {
   },
 };
 
+export const CROWDSTRIKE_CONSOLE_COMMANDS = {
+  runscript: {
+    args: {
+      raw: {
+        about: i18n.translate(
+          'xpack.securitySolution.crowdStrikeConsoleCommands.runscript.args.raw.about',
+          {
+            defaultMessage: 'Raw script content',
+          }
+        ),
+      },
+      cloudFile: {
+        about: i18n.translate(
+          'xpack.securitySolution.crowdStrikeConsoleCommands.runscript.args.cloudFile.about',
+          {
+            defaultMessage: 'Script name in cloud storage',
```

Comment:
Q: Should be file name I think? Unless script name and file name are the same?

```suggestion
            defaultMessage: 'Script file name in cloud storage',
```

Reply:
Good question. So scripts are on the Cloud, files are on a host. Again I copied this over from CrowdStrike console. But I am happy to adjust if you prefer